src/icons/fileIcons.ts

```diff
@@ -2482,5 +2482,28 @@ export const fileIcons: FileIcons = {
       ],
     },
     { name: 'apps-script', fileExtensions: ['gs'] },
+    {
+      name: 'kubernetes',
+      fileNames: [
+        'deployment.yml',
+        'deployment.yaml',
```

Comment:
Could you please only assign file names which are --> only <-- related to kubernetes. E.g. a file called deployment.yaml could be in use with soo many other projects out there which might not be related to kubernetes but to something else. And if we show a kubernetes sign all the time it's irritating.

Reply:
Uh make sense.

Reply:
@PKief I removed unnecessary files.